code/_onclick/hud/new_player.dm

```diff
@@ -256,13 +280,56 @@
 	set_button_status(TRUE)
 	UnregisterSignal(SSticker, COMSIG_TICKER_ENTER_PREGAME)
 
-/atom/movable/screen/lobby/button/settings
+//Subtype the bottom buttons away so the collapse/expand shutter goes behind them
+/atom/movable/screen/lobby/button/bottom
+	layer = LOBBY_BOTTOM_BUTTON_LAYER
 	icon = 'icons/hud/lobby/bottom_buttons.dmi'
+	always_shown = TRUE
+
+/atom/movable/screen/lobby/button/bottom/Initialize(mapload)
+	. = ..()
+	if(hud)
+		RegisterSignal(hud, COMSIG_HUD_LOBBY_COLLAPSED, PROC_REF(collapse_button))
+		RegisterSignal(hud, COMSIG_HUD_LOBBY_EXPANDED, PROC_REF(expand_button))
+
+/atom/movable/screen/lobby/button/bottom/Destroy()
+	. = ..()
+	if(hud)
+		UnregisterSignal(hud, list(COMSIG_HUD_LOBBY_COLLAPSED, COMSIG_HUD_LOBBY_EXPANDED))
+
+///Animates moving the button off-screen, disabling it
+/atom/movable/screen/lobby/button/bottom/proc/collapse_button()
+	SIGNAL_HANDLER
+	var/matrix/transformation = matrix()
+	transform = transformation
+	//wait for the shutter to come down
+	animate(src, transform = transformation, time = SHUTTER_MOVEMENT_DURATION + SHUTTER_WAIT_DURATION)
```

Comment:
still kinda confused about this why does the transformation var exist exactly?

Reply:
couldn't you just like, use transform? procs like Translate() don't modify the transform var, they instead create a copy of it, so there's no need to have a second var like this

Reply:
funny thing is that I've accidentally stumbled upon `transform` working perfectly fine but I cleaned it up in later with `transformation` lmao
https://github.com/tgstation/tgstation/pull/76443/commits/9735e710e838bb6be51edc9c03dbe0e9688a8a48